extensions/exposition/features/steps/IdP.ts

```diff
@@ -117,4 +117,31 @@ export class IdP {
 
     this.captures.set(`${user}.id_token`, idToken)
   }
+
+  @given('the IDP symmetric token for {word} is issued with following secret:')
+  public async issueSymmetricToken (user: string, secret: string): Promise<void> {
+    console.log('Sym token for %s with secret "%s"', user, secret)
+
+    const jwt = [
+      {
+        typ: 'JWT',
+        alg: 'HS256'
+      },
+      {
+        iss: IdP.issuer,
+        sub: `${user}-mock-id`,
+        aud: 'test',
+        iat: Math.floor(Date.now() / 1000),
+        exp: Math.floor((Date.now() + 1000 * 60 * 5) / 1000)
+      }
+    ]
+      .map((v) => Buffer.from(JSON.stringify(v)).toString('base64url'))
+      .join('.')
+
+    const signature = crypto.createHmac('sha256', secret).update(jwt).digest('base64url')
```

Comment:
This should be resolved from a map of symmetric algorithms.

Reply:
Fixed at https://github.com/toa-io/toa/pull/522/commits/122fd4ddb6d04fcded77746de0daf73d02e13f91